### PR TITLE
introduce caseType and jurisdiction in searchForCompletable

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5440

### Change description ###

Search for completable jurisdiction, caseType

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: api/workAllocation2/index.ts [hunk 1]
```diff
@@ -389,6 +391,7 @@ export async function postTaskSearchForCompletable(req: EnhancedRequest, res: Re
       'case_type': req.body.searchRequest.caseTypeId,
       'event_id': req.body.searchRequest.eventId,
     };
+    console.log('ravi reqBody', reqBody)
```
Comment (johnbenjamin-hmcts) [inline api/workAllocation2/index.ts:394]: Can we remove console.log statement please?

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved with minor comment

File: api/workAllocation2/index.ts (hunk 1 above)
Comment (connorpgpmcelroy) [inline api/workAllocation2/index.ts:394]: Do not think we need this?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor change

File: api/workAllocation2/index.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/workAllocation2/index.ts:394]: yes

Comment (udaydenduluri33) [reply api/workAllocation2/index.ts:394]: no

Comment (udaydenduluri33) [reply api/workAllocation2/index.ts]: Removed it

Comment (udaydenduluri33) [reply api/workAllocation2/index.ts]: Remove

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request merged**